### PR TITLE
[ios] Added search categories to Spotlight.

### PR DESCRIPTION
**Pull request by igrechuhin**
https://jira.mail.ru/browse/MAPSME-1774

### CODE REVIEW

**Comment by igrechuhin** [comment]
@ygorshenin @rokuz @VladiMihaylenko PTAL

**Comment by igrechuhin** [comment]
DUDE JTTP

**Comment by rokuz** [comment]
LGTM

**Comment by VladiMihaylenko** [comment]
LGTM

**Comment by ygorshenin** [comment]
Reviewing...

File: iphone/Maps/Classes/CustomViews/MapViewControls/Search/TabbedView/CategoriesTab/MWMSearchCategoriesManager.mm [hunk 1]
```diff
@@ -1,6 +1,7 @@
-#import "Macros.h"
 #import "MWMSearchCategoriesManager.h"
+#import "AppInfo.h"
```
Comment (ygorshenin) [inline iphone/Maps/Classes/CustomViews/MapViewControls/Search/TabbedView/CategoriesTab/MWMSearchCategoriesManager.mm]: Sort, please.

**Comment by ygorshenin** [comment]
LGTM

**Pull request merged**